### ISSUE TITLE
If creating new resource fails when form has vertical scrolling the error is not visible

### ISSUE THREAD

**Issue by dgolovin** [containers/podman-desktop#4229]
### Bug description

The form below just failed to create podman VM and I am wondering what happened.

![image](https://github.com/containers/podman-desktop/assets/620330/9ef75225-42b4-4a7d-94bc-3b2d18c3fc51)

I have to scroll the form up to see the error.

![image](https://github.com/containers/podman-desktop/assets/620330/5f0b249d-fe34-4bf9-94f5-88df2023bac0)

The error message `Error: [object Object]` is reported here #4230.

### Operating system

macOS Sonoma 14.0

### Installation Method

Other

### Version

next (development version)

**Comment by deboer-tim** [comment]
Progress is shown up there too, so for a second yesterday I thought it hadn't done anything when I clicked Create.

A simple solution would be to auto-scroll to the top when we click Create, but I don't love it when forms move themselves. Alternately we could move progress/error/logs to the bottom, have some kind of floating control, or clear the properties.

FYI to @mairin, I've added this to the next UX discussion.
